openpype/tools/publisher/window.py

```diff
@@ -879,6 +900,7 @@ def _on_publish_finished_change(self, event):
             self._comment_input.setText("")
 
     def _on_publish_stop(self):
+        self._make_sure_on_top()
```

Comment:
Oh boy, someone is going to get very very annoyed by this. We have quite some cases where a publish might be running on the background. If I understand correctly this will push it forward and take focus.

Or does it only do so if you have at least the UI/parent window active?

Reply:
In Nuke and Maya it only worked when the DCC was active. That's why the test notest are asking for other DCCs.